frontend/public/components/modals/delete-namespace-modal.tsx

```diff
@@ -46,29 +48,36 @@ export const DeleteNamespaceModal: React.FC<DeleteNamespaceModalProps> = ({
   return (
     <form onSubmit={onSubmit} name="form" className="modal-content ">
       <ModalTitle className="modal-header">
-        <YellowExclamationTriangleIcon className="co-icon-space-r" /> Delete {kind.label}?
+        <YellowExclamationTriangleIcon className="co-icon-space-r" />{' '}
+        {t('public~Delete {{label}}?', { label: kind.label })}
```

Comment:
We should internationalize what we're deleting here. The consensus was to hard-code the model instead of using the kind label, but if that's not possible, we should use `kind.labelKey` instead. (`kind.labelKey` has the internationalized version of the kind.)

Reply:
updated